pkg/meta/utils.go

```diff
@@ -385,14 +386,23 @@ func FastGetSummary(r Meta, ctx Context, inode Ino, summary *Summary, recursive
 	return 0
 }
 
-func fastGetSummary(r Meta, ctx Context, inode Ino, summary *Summary, recursive bool) syscall.Errno {
-	space, _, err := r.GetDirStat(ctx, inode)
+func fastGetSummary(r Meta, ctx Context, inode Ino, attr *Attr, summary *Summary, recursive bool) syscall.Errno {
```

Comment:
only the attribute of the top directory is useful, we can ignore it.

Reply:
Ignored